.github/workflows/test.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: Test
+
+on: workflow_dispatch
+
+jobs:
+  smoke: 
+    continue-on-error: true
+    strategy:
+      matrix:
+        file_system: ['ext4', 'zfs']
+        tpm: [true, false]
+    name: Smoke tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get code
+        uses: actions/checkout@v3
+      - name: Run Smoke tests
+        uses: ./.github/actions/run-eden-test
+        with:
+          file_system: ${{ matrix.file_system }}
+          tpm_enabled: ${{ matrix.tpm }}
+          suite: "smoke.tests.txt"
+
+  networking: 
+    name: Networking test suite
+    needs: smoke
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Get code
+        uses: actions/checkout@v3
+      - name: Run Networking tests
+        uses: ./.github/actions/run-eden-test
+        with:
+          file_system: "ext4"
+          tpm_enabled: false
```

Comment:
I wonder if we should make `tpm_enabled` as enabled for suites that do not use matrix here. I believe that most customer devices do have TPM module.